config/locales/hyrax.en.yml

```diff
@@ -1132,17 +1142,21 @@ en:
       defaults:
         admin_set_id: Administrative Set
         based_near: Location
+        contributor: Contributor
         creator: Creator
         date_created: Date Created
-        description: Description
+        description: Abstract or Summary
```

Comment:
Change this line back to "Description" for this and all other languages

Reply:
Changed all lines back to "Description" for all languages.